### DISCUSSION TITLE
PBS Password

### DISCUSSION THREAD

**Discussion by totalitarian** [tteck/Proxmox#3926] [Help Needed] [answered]
Just installed PBS, what is the default username and password?

**Comment by tteck** [comment]
If it's asking for credentials that you didn't create, the script didn't finish successfully.

It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by totalitarian** [reply]
This is the install log

Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 103
Using Hostname: pbs
Using Disk Size: 10GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a PBS LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 103 was successfully created.
 ✓ Started LXC Container
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.0.160 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 20.26.156.215
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Proxmox Backup Server
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

PBS should be reachable by going to the following URL.
         http://192.168.0.160:8007

**Reply by tteck** [reply]
**Please read the instructions!**

![Screenshot 2024-10-19 8 32 44 AM](https://github.com/user-attachments/assets/d79fff0e-013e-4efa-964a-0afe2cbe93aa)

**Reply by tteck** [reply, marked as answer]
The PBS LXC is a little different than other LXCs.
